### PR TITLE
Edit some missing attributes

### PR DESCRIPTION
**Pull request by amirhoseinshahabnia**
Adds updateAttributes to the CheckoutResource interface
Adds webUrl

Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## shopify-buy ([<kbd>unpkg</kbd>](https://unpkg.com/browse/shopify-buy@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'shopify-buy'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1e72af33b096646df286de90de7484c19ce821c7
</p>

File: types/shopify-buy/index.d.ts
Comment (MaciekBaron) [inline types/shopify-buy/index.d.ts:104]: I don't think this comment is necessary

File: types/shopify-buy/index.d.ts
Comment (MaciekBaron) [inline types/shopify-buy/index.d.ts:342]: This comment also seems unnecessary

File: types/shopify-buy/index.d.ts
Comment (MaciekBaron) [inline types/shopify-buy/index.d.ts:289]: Would it better to do `key?: string`?

File: types/shopify-buy/index.d.ts
Comment (amirhoseinshahabnia) [reply types/shopify-buy/index.d.ts:104]: agreed!

File: types/shopify-buy/index.d.ts
Comment (amirhoseinshahabnia) [reply types/shopify-buy/index.d.ts:342]: Agreed!

File: types/shopify-buy/index.d.ts
Comment (amirhoseinshahabnia) [reply types/shopify-buy/index.d.ts:289]: Yes, it will be better.

**Comment by MaciekBaron** [comment]
Hi @amirhoseinshahabnia I see you resolved the comments but I don't see changes in the code, did you forget to push? Thanks!

**Comment by MaciekBaron** [comment]
There seem to be some linting issues, I think your asterisk is not aligned.

**Comment by gabritto** [comment]
Let's wait for an approval from @MaciekBaron or another owner.

File: types/shopify-buy/index.d.ts
Comment (chrisworman-pela) [reply types/shopify-buy/index.d.ts]: Can you reference some documentation that indicates that custom attributes can have optional keys?

**Comment by chrisworman-pela** [comment]
I'll approve if someone can point me to the documentation that shows that custom attributes can have optional keys :)

**Comment by amirhoseinshahabnia** [comment]
You are right, it cannot have optional key, I will fix it now

**Comment by MaciekBaron** [comment]
LGTM

**Review by MaciekBaron — APPROVED** [review]

**Review by chrisworman-pela — APPROVED** [review]

**Review by gabritto — APPROVED** [review]

**Pull request merged**
